### PR TITLE
Cleaning codes

### PR DESCRIPTION
**Pull request by riseshia**
- Add some description
- Remove IO.inspect
- Extract type, resolve codes to another file

please review this @marocchino XD

### CODE REVIEW

File: lib/resolver.ex [hunk 1]
```diff
@@ -0,0 +1,49 @@
+defmodule Gannbaruzoi.Resolver do
```
Comment (marocchino) [inline lib/resolver.ex]: I think we can separate the files here.
- lib/resolver/task
- lib/resolver/log

File: web/models/log.ex [hunk 2]
```diff
@@ -15,8 +15,8 @@ defmodule Gannbaruzoi.Log do
   end
 
   def first_of(task_id) do
-    Repo.get Task, task_id
-    |> Ecto.assoc(:logs)
+    task = Repo.get Task, task_id
```
Comment (marocchino) [inline web/models/log.ex]: Is it make query once? Or twice?
If twice, can we reduce actually query?

Comment (riseshia) [reply web/models/log.ex]:
```
[debug] QUERY OK source="tasks" db=6.1ms decode=2.0ms
SELECT t0."id", t0."description", t0."estimated_size", t0."type", t0."status", t0."user_id", t0."inserted_at", t0."updated_at" FROM "tasks" AS t0 WHERE (t0."id" = $1) [1]
[debug] QUERY OK source="logs" db=7.6ms
SELECT l0."id", l0."task_id", l0."inserted_at", l0."updated_at" FROM "logs" AS l0 WHERE (l0."task_id" = $1) ORDER BY l0."id" LIMIT 1 [1]
[d
```

Yes, it send 2 queries, let me think about it XD

Comment (riseshia) [reply web/models/log.ex]: `Repo.one from l in __MODULE__, where: l.task_id == ^task_id, limit: 1`
fixed with this!

this generates,
```
SELECT l0."id", l0."task_id", l0."inserted_at", l0."updated_at" FROM "logs" AS l0 WHERE (l0."task_id" = $1) LIMIT 1 [1]
```

**Pull request merged**